server/static/dashboard/replay.js

```diff
@@ -0,0 +1,105 @@
+var connection = new window.sharedb.Connection(new WebSocket(shareURL));
+
+$('#startReplay').click(e => {
```

Comment:
Need to pick a version of JavaScript. If new, e.g. use `let`. If old, e.g. no `=>`.